package_python_function/packager.py

```diff
@@ -40,14 +46,32 @@ def package(self) -> None:
     def zip_all_dependencies(self, target_path: Path) -> None:
         logger.info(f"Zipping to {target_path}...")
 
-        with zipfile.ZipFile(target_path, 'w', zipfile.ZIP_DEFLATED) as zip_file:
+        def date_time() -> Tuple[int, int, int, int, int, int]:
+            """Returns date_time value used to force overwrite on all ZipInfo objects. Defaults to
+            1980-01-01 00:00:00. You can set this with the environment variable SOURCE_DATE_EPOCH as an
+            integer value representing seconds since Epoch.
+            """
+            source_date_epoch = os.environ.get("SOURCE_DATE_EPOCH", None)
+            if source_date_epoch is not None:
+                return time.gmtime(int(source_date_epoch))[:6]
+            return (1980, 1, 1, 0, 0, 0)
+
+        with zipfile.ZipFile(target_path, "w", zipfile.ZIP_DEFLATED) as zip_file:
+
             def zip_dir(path: Path) -> None:
                 for item in path.iterdir():
                     if item.is_dir():
                         zip_dir(item)
                     else:
+                        zinfo = zipfile.ZipInfo.from_file(item)
```

Comment:
@BrandonLWhite would love your feedback here.

I initially thought I would have to do: 
```python
 zinfo = zipfile.ZipInfo.from_file(item, item.relative_to(self.input_path))
```

since `from_file` accepts an `arcname` variable that is that path inside the archive. And the original code had a `relative_to` call in it.

However, I added some tests, and they pass without the `arcname` specified. 

So I think we're good here? Or is there an edge case I'm missing?

Reply:
So long as the path that gets stored in the ZIP file is relative to the input_path, then it's all good.
Perhaps zip_file.open is doing a `relative_to` or similar internally since it has both paths available to it.

Reply:
That's kind of what I'm thinking as well.

Reply:
Turns out that I just needed to fix my tests